runsc/container/container.go

```diff
@@ -822,6 +827,15 @@ func (c *Container) Destroy() error {
 	return fmt.Errorf(strings.Join(errs, "\n"))
 }
 
+// NewFDMapWithOriginalFDs creates a mapping from file descriptors to files.
+func NewFDMapWithOriginalFDs(files ...*os.File) map[int]*os.File {
+	fdMap := map[int]*os.File{}
+	for _, file := range files {
+		fdMap[int(file.Fd())] = file
+	}
+	return fdMap
+}
+
```

Comment:
This is the same as `control.NewFDMapWithOriginalFDs`.  Do we need both?

Reply:
I have removed the `NewFDMapWithOriginalFDs` convenience methods now to reduce the complexity of this PR. After all, using `NewFDMap` should already be convenient enough:
https://github.com/google/gvisor/blob/feb40ac6b6479436b7d5c914dd640a43a00540a7/runsc/container/container_test.go#L82-L84